ui/ui-components/components/navigation/setupStepsNavProgressBar.tsx

```diff
@@ -58,36 +59,52 @@ export default function SetupStepsNavProgressBar({
 
   if (!shouldDisplay) return null;
 
+  if (isOnBoardingComplete) {
+    return (
+      <Row className="pt-1 px-4">
+        <Col>
+          <Link href="/setup">
+            <a>Get Started:</a>
+          </Link>
+        </Col>
+      </Row>
+    );
+  }
+
   return (
     <div
+      className="m-2 px-1 pb-3 pt-1 rounded"
       style={{
         backgroundColor: "var(--grouparoo-background-blue)",
-        width: "100%",
-        padding: 20,
-        marginTop: 10,
       }}
     >
-      <div
+      <Row
+        className="px-3"
         style={{
           fontSize: 18,
-          paddingLeft: 0,
-          color: "var(--secondary)",
-        }}
-      >
-        <Link href="/setup">
-          <a>{isOnBoardingComplete ? "Setup Complete 🎉" : "Get Started:"}</a>
-        </Link>
-      </div>
-      <div
-        style={{
-          paddingLeft: 0,
-          paddingBottom: 10,
           color: "var(--secondary)",
         }}
       >
-        {activeStep?.title}
-      </div>
-      <ProgressBar now={percentComplete} />
+        <Col>
+          <Link href="/setup">
+            <a>{isOnBoardingComplete ? "Setup Complete 🎉" : "Get Started:"}</a>
```

Comment:
If you are early returning due to `isOnBoardingComplete` above, this check isn't necessary any more.